src/fixtures/story.html

```diff
@@ -1,152 +1,202 @@
 <!DOCTYPE html>
 <html amp="🤠-invalid" lang="en">
-<!-- TODO(alanorozco): Run this template through the AMP optimizer -->
-<head>
-  <meta charset="utf-8" />
-  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
-  <script async src="https://cdn.ampproject.org/v0.js"></script>
-  <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
-  <title>AMP Story</title>
-  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1" />
-  <link rel="canonical" href="/" />
-  <style amp-custom>
-    .i-amphtml-glass-pane {
-      height: 100% !important;
-      width: 100% !important;
-      z-index: 1 !important;
-    }
-
-    .ad {
-      border: none;
-      width: 100% !important;
-      height: 100% !important;
-    }
+  <!-- TODO(alanorozco): Run this template through the AMP optimizer -->
+  <head>
+    <meta charset="utf-8" />
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script>
+      if (window.history.replaceState) {
```

Comment:
- Check for both `window.history` and `window.history.replaceState` as not to break if `history` is not available on the running platform.
- Add a comment explaining what this snippet does.

Reply:
i notice that the comments are short and concise, would this be a fitting comment for this code snippet?
<!--makes story ad start on 'page-1' (ad page)-->

Reply:
Something like "navigate story so that ad page is visible"

Reply:
Also if we keep adding patches here we can maybe move to separate file, `hacks.js` :)

Reply:
@bhassan6621  In this case the comment is just to give context. It's a little obscure, so it helps with the "why" :)